public/app/features/logs/components/log-context/LogContextButtons.tsx

```diff
@@ -1,14 +1,18 @@
 import { css } from '@emotion/css';
-import React from 'react';
+import React, { useCallback } from 'react';
 
 import { SelectableValue } from '@grafana/data';
-import { ButtonGroup, ButtonSelect, useStyles2 } from '@grafana/ui';
+import { reportInteraction } from '@grafana/runtime';
+import { ButtonGroup, ButtonSelect, InlineField, InlineFieldRow, InlineSwitch, useStyles2 } from '@grafana/ui';
 
 const getStyles = () => {
   return {
-    logSamplesButton: css`
+    inlineButtons: css`
```

Comment:
Very minor, but I don't like that the class name matches the visual properties (inline, center), and not its semanting meaning in the component, i.e., what they are styling.

Also center is not used?

Reply:
Good feedback. Changed in 536db8d5d7.